app/src/main/res/raw/infinite_level

```diff
@@ -0,0 +1,7 @@
+{
+  "baseSpeed": 30,
+  "firstObstacle": {
```

Comment:
Why do we need first obstacle?

Reply:
It can be generated. For all further obstacles, generation will be based on the previous obstacle, so the first one can either be set directly or generated using a different method.

Reply:
Removed